applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -286,17 +302,32 @@ def AddVariables(self):
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NORMAL)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.Y_WALL)
         self.main_model_part.AddNodalSolutionStepVariable(KratosCFD.Q_VALUE)
+
+        # Adding variables required for the turbulence modelling
+        if self.turbulence_model_solver is not None:
```

Comment:
I think
```suggestion
        if self.turbulence_model_solver:
```

should do the same (please check if you consider using it, just a suggestion)

Reply:
it works.... i did the change... :)